### ISSUE TITLE
Update SRComp Scorer with new UI

### ISSUE THREAD

**Issue by PeterJCLaw** [srobo/tasks#175]
The score-input UI should match the score-sheets as much as possible to reduce the chance of errors when inputting the data. The _design_ of both should therefore happen at the same time (see #109), so this ticket is about making the changes to the software rather than doing the design work.

Note that this update is also linked to and dependant upon the scoring logic as the UI needs to output data in a format which the logic can ingest (though there are existing standard protocols for much of the structure).

See https://github.com/PeterJCLaw/srcomp/wiki/Competition-Preparation#configure-game-scoring for more details.

**Comment by trickeydan** [comment]
This appears to be a duplicate of #108. Please check whether an issue exists before opening it.

I would also like to point out that this is a "should have" as we can operate a competition without this system.

**Comment by PeterJCLaw** [comment]
Aha, thanks. I'd missed that one when I went through the notification emails I got for the new tickets. For some reason I don't seem to be able to comment on that other one (nor got any notifications about it).

I disagree that this is only a should-have. SR is well past the scale where running the scoring manually is viable, meaning that we need a mechanism to convert the scores from the paper sheets into a machine readable format and have the scoring be automated (#111).
